packages/pigeon/run_tests.sh

```diff
@@ -328,9 +328,27 @@ run_formatter() {
   pub global activate flutter_plugin_tools && pub global run flutter_plugin_tools format 2>/dev/null
 }
 
+run_android_unittests() {
+  pushd $PWD
+  pub run pigeon \
```

Comment:
since we were talking about it last week, fwiw, I still think the static setup function is helpful since these parameters can be just checked in in the pigeons/android_unittests.dart file (which is helpful for teams who work on the same files but don't check in a script that does the building).